### ISSUE TITLE
IRISMAN stuck at "Scanning All Games" Screen

### ISSUE THREAD

**Issue by nathanriggs** [aldostools/IRISMAN#1]
Using newest version of Rebug, clean install from original FW. IRISMAN worked as expected for a couple of weeks, but suddenly stopped working for reasons unknown. I'd be happy to share any information I can.

**Comment by nathanriggs** [comment]
@aldostools

**Comment by aldostools** [comment]
It could be some of the new content that you added recently that have corrupted the game list.

My suggestion is that delete IRISMAN through XMB. Then re-install IRISMAN and start it **without** any external device connected. If it works, then scan the games adding the external devices one by one until you find the device that is corrupting the game list.

**Comment by nathanriggs** [comment]
Sorry it took so long to respond to this, but the problem had nothing to do with the NTFS drive. There was *something* corrupted in the PSPISO folder that was causing IRISMAN to hang. I deleted all of my games, testing each time a system's games were deleted, and once I deleted the PSP games IRISMAN began to work as usual again. 

Once I get the ps3,ps2 and psx games installed again, I'll slowly go through the psp games to try to figure out which one has caused the issue.
